sale_view_adj_sst/views/sale_view_adj.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="sale_view_order_line_filter" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_sales_order_filter" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='product_id']" position="after">
+                <field name='order_line' />
```

Comment:
```suggestion
                <field name="order_line" />
```

A tiny adjustment if you may.  We should standardize to use the double quotations wherever possible, as it can affect the searchability (you see single and double quotations used rather randomly in the standard Odoo code base, which in my opinion is not very good).

Reply:
I understand, I'll use double quotation as standard.